### ISSUE TITLE
Make BLAT work for assemblies

### ISSUE THREAD

**Issue by ifiddes** [ucsc-mus-strain-cactus/build_browser#14]

**Comment by diekhans** [comment]
This is problematic because we have to setup new blat server and
it would not be appropriate to run them on hgwdev.

**Comment by joelarmstrong** [comment]
Obviously out of scope for this project, but shouldn't it be possible to

**Comment by joelarmstrong** [comment]
Whoops, hit the wrong button.

I was going to ask, just out of curiosity, shouldn't it be technically possible to keep blat's index on disk for stuff like mice or the rarely-used genomes? I assume that most of the less popular browsers don't have blat servers because it's prohibitive to keep several GB of memory constantly in use for each genome.

**Comment by diekhans** [comment]
Yes, that is actually something I suggest to Jim at one time.  Use mmap
to just open the index.

I don't know the index access patterns, so it might very slow.

Maybe worth an experiment, but they still would not be happy
running it on hgwdev.
